src/index.html

```diff
@@ -5,9 +5,46 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="./styles/index.scss">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-logo"></div>
+      <div class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="product-code">Product code: 195434</div>
+      <div class="reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+
+        <div class="reviews__text" >
+          Reviews: 5
+        </div>
+      </div>
+      <div class="price">
+        <div class="price__text">
+          Price:
+        </div>
+        <div class="price__number">
+          $2,199
+        </div>
+      </div>
+      <a href="#" class="link">
+        <div class="buy" data-qa="hover">
```

Comment:
you don't need this div wrapper. just give same styles for link

Reply:
if i do it my buy styles will broke